packages/amplify-cli/src/extensions/amplify-helpers/show-auth-acm.ts

```diff
@@ -24,7 +25,9 @@ export function showACM(sdl: string, nodeName: string) {
     const acm = new AccessControlMatrix({ name: type.name.value, operations: MODEL_OPERATIONS, resources: fields });
     const parentAuthDirective = type.directives?.find(dir => dir.name.value === 'auth');
     if (parentAuthDirective) {
-      const authRules: AuthRule[] = getAuthDirectiveRules(new DirectiveWrapper(parentAuthDirective));
+      const authRules: AuthRule[] = getAuthDirectiveRules(new DirectiveWrapper(parentAuthDirective), false, {
```

Comment:
There is one more place where we have the same method call [here](https://github.com/aws-amplify/amplify-cli/pull/10915/files#diff-86aacfb3d4a9e3ac7d29ce386169b1c55f7eea125966104a92d81ef0318decdfR39)